identity/backend/src/main/java/io/camunda/identity/usermanagement/repository/UserRepository.java

```diff
@@ -7,23 +7,39 @@
  */
 package io.camunda.identity.usermanagement.repository;
 
-import io.camunda.identity.user.CamundaUser;
+import io.camunda.identity.usermanagement.CamundaUser;
+import io.camunda.identity.usermanagement.model.Profile;
 import java.util.List;
-import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public class UserRepository {
-  public static final String DEF_USERS_QUERY = "select username, enabled from users ";
+public interface UserRepository extends JpaRepository<Profile, Long> {
```

Comment:
I find `UserRepository` a little confusing as it's actually for the `Profile` entity and thus the `profiles` table. I'd like to suggest to call it `ProfileRepository` or alternatively think about changing the name `Profile` to `UserProfile`.

Personally being a fan of expressive names, I would like `UserProfile` much. But no strong opinion, just my 2 cents 😉 

WDYT?